src/main/scala/za/co/absa/commons/io/TempDirectory.scala

```diff
@@ -75,12 +76,12 @@ class TempDirectory private(prefix: String, suffix: String, pathOnly: Boolean) {
   def toURI: URI = path.toFile.toURI
 
   /**
-   * The TempDirectory object will be converted to String type.
-   * Automated "\ -> /" conversion will be performed to reach stable outputs across different OS.
+   * The TempDirectory object will be converted to string representation which is stable and
+   * equal on all platforms.
```

Comment:
I liked how the original specifically mentioned that _ `"\ -> /" conversion will be performed_. This is a valuable information and missing in the new version, while this is still true.

Reply:
If being picky I'd emphasize in the javadoc that it's the _last part_ of the path that is meant to be equal across platforms, not the whole path as it stats with the temp dir location that is different on platforms anyway

Reply:
ok, I don't have a strong feeling about this, it was just an opinion. I approved nonetheless.